src/main/webapp/index.html

```diff
@@ -22,6 +22,18 @@ <h1>
   <button onclick="graph.generateGraph()">Get Graph</button>
   <br />
 
+  <label>Find Node in Graph: </label>
+  <input type="text" id="node-search" placeholder="Node Name"></input>
+  <button id="search-button">Find Node</button>
+  <label id="node-error"></label>
+  <label>Find Token in Graph: </label>
+  <input type="text" id="token-search" placeholder="Token Name"></input>
+  <button id="search-token-button">Find Token</button>
```

Comment:
Maybe insert some linebreaks between the label-input pairs? It looks a bit crowded when all the fields are populated.